dvc/project.py

```diff
@@ -143,6 +143,14 @@ def _check_output_duplication(self, outs):
                         stages = [o.stage.relpath, out.stage.relpath]
                         raise OutputDuplicationError(o.path, stages)
 
+    def _check_outs_duplication(self, outs):
+        if len(outs) != len(set(outs)):
+            msg = "Output '{}' was specified more than one time"
+            f = set([x for x in outs if outs.count(x) > 1])
+            for i in f:
+                s = i + " "
+            self.logger.warn(msg.format(s))
```

Comment:
Please add something like StageOutputDuplicationError exception(similar to existing OutputDuplicationError) and raise it when duplication is detected.